src/utils/format-numbers.js

```diff
@@ -50,6 +64,22 @@ export function formatBytes(bytes: number): string {
   }
 }
 
+export function formatMicroseconds(
+  time: Microseconds,
+  sagnificantDigits: number = 2,
```

Comment:
oops: typo :) you replaced a `i` by a `a` in `significantDigits` :)

The same everywhere. Will you be able to carry the change? Or otherwise I can just do it myself and merge, as you wish :)